api/rules.md

```diff
@@ -83,13 +83,30 @@ The [Exponential Backoff Calculator](http://backoffcalculator.com/?interval=5&ra
 
 Only send an **order** or **order response** that either **is new** or **has an actual change**
 
+### Never send all orders or responses periodically
+
+Never resend **all** or **all active** orders or responses periodically.
+
 ### The order or order response should only contain new or changed lines
 
 The order or order response should only contain **order lines** that are **new or changed**. Sending an unchanged order line could trigger an unexpected line status change in Tradecloud.
 
-### Never send all orders or responses periodically
+### Arrays are limited to 100 objects.
 
-Never resend **all** or **all active** orders or responses periodically.
+The TOTAL number of objects is default limited to 100 objects per collection.
+
+Examples are:
+
+- 100 documents per order in total
```

Comment:
```suggestion
- 100 documents in total per order header
```

It's not really correct, but I think it may prevent confusion

Reply:
Updated